build.sbt

```diff
@@ -130,8 +130,11 @@ lazy val common = project
       scalaVersion,
       sbtVersion,
       git.gitHeadCommit,
-      BuildInfoKey.action("buildTime")(System.currentTimeMillis)
     ),
+    buildInfoKeys ++= {
```

Comment:
We are getting extra compilation when e.g. the style files change. I tracked this to the `BuildInfo` that reads the time on every change.
With this change you can set an env variable `SBT_IGNORE_BUILDTIME` and it will set a fixed time instead

Reply:
Nice, thank you!